docs/en/rules/Azure.APIM.AvailabilityZone.md

```diff
@@ -173,6 +176,10 @@ resource service_api_mgmt_test2_name_resource 'Microsoft.ApiManagement/service@2
 }
 ```
 
+## NOTES
+
+For developer environments, supressing the rule might make sense as enabling zone redundancy for an API Management instance requries the `Premium` SKU currently.
```

Comment:
The `NOTES` header is listed above. I like your positioning of `NOTES`, so maybe move the content from the `NOTES` above to this section so there is only one `NOTES`.

```suggestion
For developer environments, suppressing the rule might make sense as enabling zone redundancy for an API Management instance requries the `Premium` SKU currently.
```